README.md

```diff
@@ -30,6 +30,64 @@ int main() {
 }
 ```
 
+## Fetching meta data:
```

Comment:
Could you just please put this one level under `Usage Example` section? Probably even rename `Usage Example` to `Usage Examples` where both of these examples can be at the same time. Thanks for contribution though.

Reply:
Sure thank you!